src/kgcl_schema/grammar/parser.py

```diff
@@ -54,6 +58,15 @@ def parse_statement(input: str) -> Change:
 
     Return an instantiated dataclass object from model.kgcl_schema.
     """
+    regex = r'<http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\(\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+'
+    uri = re.findall(regex, input)
+    if uri:
+        # curie = curie_from_iri(uri[0].replace("<", "").replace(">",""))
+        pref, i = parse_iri(uri[0].replace("<", "").replace(">",""))
```

Comment:
There it is again - I am having long debates about the `<>` with @joeflack4 as well. This URI hacking smells bad, like the input is not properly sanitised. Why is this necessary, where does the `<>` come from?

Reply:
Same with the regex above. you seem to _expect_ uris with `<>`..

Reply:
That's what is provided in the command itself. I think there is documentation about adding the `<>` for URIs

Reply:
Addressed this. Removed all `<>` references for URIs in KGCL.